docs/jwst/data_products/product_types.rst

```diff
@@ -1,81 +1,131 @@
 Data Product Types
 ------------------
-The following table contains a list of all data product types, as given by their file name suffix,
-the pipeline and stage of processing that creates each type, and an indication of whether the file naming
-is exposure-based or source-based. The product name suffixes are active links to detailed descriptions
-in the following sections.
+The following tables contain a list of all data product types, as given by their file name suffix. There is one table per stage of processing.
```

Comment:
"tables" is plural, hence "list" also needs to be plural: The following tables contain lists of all data product ..."

Reply:
Done